README.md

```diff
@@ -8,6 +8,12 @@ pylabrad is an interface to the LabRAD system in python with support for both cl
 For general information about the LabRAD system/protocol see [the labrad repository](https://github.com/labrad/labrad) and associated [wiki](https://github.com/labrad/labrad/wiki).
 For help getting started and understanding pylabrad take a look at the [wiki](https://github.com/labrad/pylabrad/wiki).
 
+## Manager Compatibility
+
+Compatibility with the Sourceforge manager and explorer is no longer supported.
```

Comment:
`As of version 0.96.0, pylabrad is no longer compatible with the Delphi labrad manager available from Sourceforge.`